### DISCUSSION TITLE
Tiling window remembers dimensions

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qtile/qtile#2667] [Q&A]
Is there a layout,or any other way to have a window remember it's dimensions?
Ex. 
I open `vifm` as a small column on the side , and on the other side zathura reading a pdf. I close zathura to open another pdf through vifm. I don't want to have to readjust the size of the vifm window again.

Is there a way to do that? I think spectrwm has a similar feature.

**Comment by m-col** [comment]
Perhaps you could try setting this up at the level of the layout, for example by using the `RatioTile` layout and specifying a ratio between the windows, then use that layout for when you're using vifm+zathura

**Reply by LamprosPitsillos** [reply]
I tried messing with Ratio tiling but i got confused. The documentation isn't really clear.
Can you please elaborate more on how i could do this?
Thank you
